### DISCUSSION TITLE
HTML files in static paths not being copied over

### DISCUSSION THREAD

**Discussion by scpedicini** [getpelican/pelican#3407] [Q&A] [answered]
I checked through the documentation and noticed this: 

*In the default configuration, all files with a valid content file suffix (.html, .rst, .md, …) get processed by the article and page generators before the static generator. This is avoided by altering the *_EXCLUDE settings appropriately.*

The issue is I have a STATIC_PATH set to a folder called "other", I need every single file and folder within that directory to be copied over in an unaltered state to the corresponding output folder when I build, and unfortunately it copies over every single file except for the HTML files.

Is there anyway I can disable the rst/markdown/html preprocessor for explicitly designated static paths?

**Comment by scpedicini** [comment]
On further reading of the documentation, it almost sounds like I can add the path of `other` into both the `article exclude` and `page exclude` and that should solve the issue.

**Comment by justinmayer** [comment, marked as answer]
Do you have any HTML files in your _source_ `content`? If not, have a look at the `READERS` setting in the Settings section of the documentation. To wit:

    READERS = {'html': None}

**Reply by scpedicini** [reply]
@justinmayer oh I see. yeah all my "content" files are standard markdown. It's only the static folder that has some HTML files. This requirement came up recently because I build a lot of "interactive embeds" and the easiest way to integrate them into my pelican site is to keep them completely hands off from pelican and link them in the content level markdown note using an iframe. 

Additionally, it looks like adding the `other` folder to my exclusions (page, article) also seems to work~

Thanks!
